src/environments/environment.ts

```diff
@@ -3,7 +3,8 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
-  production: false
+  production: false,
+  baseUrl: 'https://jbossnewserver.herokuapp.com'
 };
```

Comment:
Did you test this base URL? '/' is missing

Reply:
No... you didn’t test this URL! I found it in the issues section where it was stated as a new server and I used it!

Reply:
Done adding '/' ✅